docs/reference/signals.rst

```diff
@@ -82,3 +82,17 @@ This signal is emitted from a ``Page`` when the page is unpublished.
 :sender: The page ``class``
 :instance: The specific ``Page`` instance.
 :kwargs: Any other arguments passed to ``page_unpublished.send()``
+
+
+page_moved
+----------
+
+.. versionadded:: 2.5
+
+This signal is emitted from a ``Page`` when the page is moved to a different branch of the page tree, or reordered within the same branch.
+
+:sender: The page ``class``
+:instance: The specific ``Page`` instance.
+:position_page: The ``Page`` instance used in conjunction with ``position`` to specify the new tree position for ``instance`` (the intended parent or sibling page).
```

Comment:
Thanks! I actually went for the terminology used in the view here, rather than by the `move()` method itself (so, there is already some consistency). If you can confirm that you'd be happier with `target`, I will happily change :)

Reply:
We probably should be matching the terminology with the ``move`` method since that's already public API. I'm happier with ``target``, thanks!